x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -32,22 +32,23 @@ import { createEndpointHost } from '../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../tasks/delete_all_endpoint_data';
 import { enableAllPolicyProtections } from '../../tasks/endpoint_policy';
 
-// Failing: See https://github.com/elastic/kibana/issues/169903
-describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
+describe('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
   let indexedPolicy: IndexedFleetEndpointPolicyResponse;
   let policy: PolicyData;
   let createdHost: CreateAndEnrollEndpointHostResponse;
+  let beforeChain: Cypress.Chainable;
 
   before(() => {
-    getEndpointIntegrationVersion().then((version) => {
-      createAgentPolicyTask(version).then((data) => {
+    beforeChain = getEndpointIntegrationVersion().then((version) => {
```

Comment:
Hmmm. This is an interesting approach and one that I did not think we needed for Cypress. What led you to implement this?

Several of our tests use the code for standing up a real endpoint VM and they are not having issues, so I'm not sure this is a solution for whatever problem you encountered.